src/components/internal/Popover.tsx

```diff
@@ -29,6 +29,10 @@ export function Popover(props: PopoverProps) {
           return true;
         }
 
+        // Do not close the Modal if the user is interacting with the Tribute mentions dropdown (via RichTextField) or with another 3rd party dialog (such as a lightbox) on top of it.
+        return !(e.closest(".tribute-container") || e.closest("[role='dialog']") || e.closest("[role='alert']"));
+
+
         onClose();
         return false;
```

Comment:
The addition here makes it so `onClose` isn't called. Does this still work as expected?

Reply:
(We're now returning before `onClose` can be called or the `return false` can be called.